src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -18,6 +18,7 @@
 from typing import Optional, Tuple, Union
 
 import torch
+import torch.fx
```

Comment:
This might need to be gated behind an `if is_torch_fx_available():` or is the module already present in torch 1.9 (minimum version is 1.10 in the imports file, but it may be because we use features in 1.10+ for the torch fx conversion script)?

Reply:
if it's not available `@torch.fx.wrap` will fail. I tested with pt-1.9 and it works with the current version of this PR.

Reply:
tested with 1.9 and 1.10 as well